fury/tests/test_actors.py

```diff
@@ -1365,3 +1365,20 @@ def test_billboard_actor(interactive=False):
     scene.add(actor.axes())
     if interactive:
         window.show(scene)
+
+
+def test_sdf_actor(interactive=False):
+    scene = window.Scene()
+    scene.background((1, 1, 1))
+    centers = np.array([[2, 0, 0], [0, 2, 0], [0, 0, 0]])
+    colors = np.array([[255, 0, 0], [0, 255, 0], [0, 0, 255]])
+    directions = np.array([[0, 1, 0], [1, 0, 0], [0, 0, 1]])
+    scale = [1, 2, 3]
+    primitive = ['sphere','sphere','torus']
+
+    sdf_actor = actor.sdf(centers, directions,
+                          colors, primitive, scale)
+    scene.add(sdf_actor)
+    scene.add(actor.axes())
+    if interactive:
+        window.show(scene)
```

Comment:
look at other tests to complete this test. You need to do a `snapshot`and check if the desired number of objects is present. look [here](https://github.com/fury-gl/fury/blob/master/fury/tests/test_actors.py#L858) and [here](https://github.com/fury-gl/fury/blob/master/fury/tests/test_actors.py#L860)